triggers/service/authentication.md

```diff
@@ -41,12 +41,13 @@ all that much, they just need to be fixed once).
 
 2. /login If /auth returned unauthorized, the trigger service will
    redirect users to this. The parameters will include the requested
-   claims as well as the URL that the users tried to access originally
-   (e.g., the endpoint to start a trigger). This will start an auth
-   flow, e.g., an OAuth2 authorization code grant ending with the auth
-   service setting a cookie with the access and refresh token and
-   redirecting to the original URL. At this point, the request to
-   /auth will succeed (based on the cookie).
+   claims as well as a callback URL. This will start an auth flow,
```

Comment:
```suggestion
   claims as well as a callback URL (note that this is not the OAuth2 callback url but a callback URL on the trigger service). This will start an auth flow,
```